packages/react/src/tests/Global.ts

```diff
@@ -0,0 +1,3 @@
+export const setup = () => {
+  process.env.TZ = "UTC";
```

Comment:
Better to have a common tz between all test than a random one.
I don't know if UTC is fine or if we should pick a other one in purpose.
A tz with a delta can reveal some bug we want to catch.

Reply:
Using another timezone would lead to have it mind to every test that complefixy.